### ISSUE TITLE
Add Multiple Email Addresses

### ISSUE THREAD

**Issue by sockpapi** [elie222/inbox-zero#27]
This tool would be very effective for me (and many others) if we were able to add multiple email addresses in 1 account and management them as well. Like a unibox

**Comment by elie222** [comment]
Agree that would be an awesome feature. I also have multiple inboxes. Would love to get this added in the next month.

**Comment by mbergwall2222** [comment]
@elie222 Is this a WIP? If not, I would love to attempt it.

**Comment by elie222** [comment]
> @elie222 Is this a WIP? If not, I would love to attempt it.

I don't think so. But join our Discord to make sure there's no overlap: getinboxzero.com/discord

**Comment by Debdut0122** [comment]
Hi @elie222 , is this still available? I would like to attempt!

**Comment by elie222** [comment]
Hey, it isn't. I'm working on it now. Quite a large PR. Should be live by next week.
